README.md

```diff
@@ -2,6 +2,8 @@
 
 C++ and Python examples of using MoveIt 2 for planning motions that are executed inside ~~Ignition~~ Gazebo simulation environment. These examples make use of [ros2_control](https://github.com/ros-controls/ros2_control) via [gz_ros2_control](https://github.com/ros-controls/gz_ros2_control).
 
+The interaction among different ROS 2, MoveIt 2 and Gazebo components is further documented in [docs/README.md](./docs/README.md), alongside suggestions for implementing a similar setup on your custom robot.
```

Comment:
I added this sentence to the main README to increase the visibility of the documentation you worked on. Please, let me know if that is okay with you.